utils/sendEmail.js

```diff
@@ -0,0 +1,93 @@
+
+const sgMail = require('@sendgrid/mail');
+require('dotenv').config();
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const sendVerificationToken = async (mail, token) => {
```

Comment:
##### Generyczna funkcja wysyłania email
Dobra robota z dotychczasową implementacją! Jednakże, warto by było uczynić funkcję odpowiedzialną za wysyłanie wiadomości bardziej generyczną. To pozwoli na jej łatwiejsze wykorzystanie w różnych częściach aplikacji, a także ułatwi modyfikację i rozbudowę w przyszłości.
Spróbuj dodać parametry do funkcji, takie jak adresat, temat, treść wiadomości i inne opcje konfiguracyjne, które mogą być potrzebne w zależności od kontekstu użycia. Dzięki temu będziesz mogła używać tej samej funkcji do wysyłania różnych rodzajów wiadomości bez konieczności tworzenia nowej funkcji za każdym razem, gdy potrzebujesz wysłać coś innego.